doc/user/content/sql/functions/_index.md

```diff
@@ -102,6 +102,10 @@ Operator | Computes
 Operator | Computes
 ---------|---------
 <code>&vert;&vert;</code> | Concatenation
+`~~` | Matches LIKE pattern case sensitively, using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
+`~~*` | Matches LIKE pattern case insensitively (ILIKE), using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
+`!~~` | Does not match LIKE pattern case sensitively, using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
```

Comment:
```suggestion
`!~~` | Matches NOT LIKE pattern case sensitively, using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
```

Reply:
Yeah, these were also bothering me a bit... I've changed all of them now.